src/gui_dwrite.h

```diff
@@ -13,41 +13,45 @@
 #ifndef GUI_DWRITE_H
 #define GUI_DWRITE_H
 
+// clang-format off
```

Comment:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#indentexternblock
`IndentExternBlock: NoIndent` looks better than "clang-format off".

Reply:
Oh, I didn't know that ExternBlocks should be indented. Will fix it!